CONTRIBUTING.md

```diff
@@ -0,0 +1,84 @@
+# Contributing Guidelines
+
+Thank you for your interest in contributing to our project. Whether it's a bug report, new example, or correction, we greatly value feedback and contributions from our community.
+
+Please read through this document before submitting any issues or pull requests to ensure we have all the necessary
+information to effectively respond to your bug report or contribution.
+
+
+## Table of Contents
+
+* [Report Bugs](#report-bugs)
+* [Contribute via Pull Requests (PRs)](#contribute-via-pull-requests-prs)
+  * [Pull Down the Code](#pull-down-the-code)
+  * [Making your Changes](#making-your-changes)
+  * [Send a Pull Request](#send-a-pull-request)
+* [Code of Conduct](#code-of-conduct)
+* [Security Issue Notifications](#security-issue-notifications)
+* [Licensing](#licensing)
+
+## Report Bugs
+
+We welcome you to use the GitHub issue tracker to report bugs.
+
+When filing an issue, please check [existing open](https://github.com/aws/amazon-braket-examples/issues) and [recently closed](https://github.com/aws/amazon-braket-examples/issues?q=is%3Aissue+is%3Aclosed) issues to make sure somebody else hasn't already
+reported the issue. Please try to include as much information as you can. Details like these are incredibly useful:
+
+* A reproducible test case or series of steps.
+* The version of our code being used.
+* Any modifications you've made relevant to the bug.
+* A description of your environment or deployment.
+
+
+## Contribute via Pull Requests (PRs)
+
+Contributions via pull requests are much appreciated.
+
+Before sending us a pull request, please ensure that:
+
+* You are working against the latest source on the *main* branch.
+* You check the existing open and recently merged pull requests to make sure someone else hasn't already addressed the problem or created a similar example.
+* You open an issue to discuss any significant work - we would hate for your time to be wasted.
+
+
+### Pull Down the Code
+
+1. If you do not already have one, create a GitHub account by following the prompts at [Join Github](https://github.com/join).
+1. Create a fork of this repository on GitHub. You should end up with a fork at `https://github.com/<username>/amazon-braket-examples`.
+   1. Follow the instructions at [Fork a Repo](https://help.github.com/en/articles/fork-a-repo) to fork a GitHub repository.
+1. Clone your fork of the repository: `git clone https://github.com/<username>/amazon-braket-examples` where `<username>` is your github username.
+
+
+### Making your changes
+When you make a contribute please ensure that you
+1. Follow the existing flow of a notebook ([example](https://github.com/aws/amazon-braket-examples/blob/main/examples/getting_started/1_Running_quantum_circuits_on_simulators.ipynb)).
+1. Do not duplicate existing information but refer to other examples as relevant.
+1. Do not rely on dependencies unless they're common Open Source licenses such as Apache 2.0 or BSD.
```

Comment:
If possible, it would be helpful to list/link to the exact set of acceptable OS licenses. As a contributor, I could be confused on whether it qualifies as a common OS license.

Reply:
I reworded "Only have Open Source licensed dependencies in your example.". Owners of the repo will need to verify the compliance here. I do not believe there's an explicit list that can be shared.